elliott/elliottlib/cli/find_bugs_second_fix_cli.py

```diff
@@ -0,0 +1,101 @@
+import traceback
+from typing import Dict, List
+
+import click
+from artcommonlib import logutil
+
+from elliottlib import Runtime
+from elliottlib.bzutil import Bug, BugTracker, is_first_fix_any
+from elliottlib.cli.common import cli
+from elliottlib.cli.find_bugs_sweep_cli import FindBugsMode
+
+LOGGER = logutil.get_logger(__name__)
+
+
+class FindBugsSecondFix(FindBugsMode):
+    def __init__(self):
+        super().__init__(
+            status={'MODIFIED', 'VERIFIED', 'ON_QA'},
+            cve_only=True,
+        )
+
+
+@cli.command("find-bugs:second-fix", short_help="Closes trackers if they are NOT first fix when branch is pre-release")
+@click.option("--noop", "--dry-run", is_flag=True, default=False, help="Don't change anything")
+@click.pass_obj
+def find_bugs_second_fix_cli(runtime: Runtime, noop):
+    """Checks if CVE trackers in a pre-release branch [before GA] are not 'first fix',
+    and if so, they will be closed
+    \b
+
+        $ elliott -g openshift-4.Y find-bugs:second-fix
+    """
+    runtime.initialize()
+    find_bugs_obj = FindBugsSecondFix()
+    for b in [runtime.get_bug_tracker('jira')]:
+        find_bugs_second_fix(runtime, find_bugs_obj, noop, b)
+
+
+def find_bugs_second_fix(runtime, find_bugs_obj, noop, bug_tracker):
+    if runtime.group_config.software_lifecycle.phase == 'pre-release':
+        LOGGER.info(
+            f"Software lifecycle is {runtime.group_config.software_lifecycle.phase}. Performing non-first-fix action ..."
+        )
+        major_version, minor_version = runtime.get_major_minor()
+        trackers = find_bugs_obj.search(bug_tracker_obj=bug_tracker, verbose=runtime.debug)
+        flaw_bug_tracker = runtime.get_bug_tracker('bugzilla')
+        brew_api = runtime.build_retrying_koji_client()
+        tracker_flaws, not_first_fix_flaw_bugs = get_flaws(flaw_bug_tracker, trackers, brew_api, strict=False)
+        # Extract Bugzilla IDs from not_first_flaw_bugs into a set for efficient lookup
+        not_first_flaw_bug_ids_set = {bug.id for bug in not_first_fix_flaw_bugs}
+        # Filter tracker_flaws based on these Bugzilla IDs
+        matching_tracker_keys_list = []
+
+        for tracker_id, bugzilla_ids_list in tracker_flaws.items():
+            if any(bz_id in not_first_flaw_bug_ids_set for bz_id in bugzilla_ids_list):
+                matching_tracker_keys_list.append(tracker_id)
+
+        new_bug_tracker = runtime.get_bug_tracker('jira')
```

Comment:
don't need this.. we can use `bug_tracker`

Reply:
Done